src/com/amazon/ion/benchmark/ReadGeneralConstraints.java

```diff
@@ -14,6 +14,7 @@
 
 import java.io.FileOutputStream;
 import java.io.OutputStream;
+import java.util.Random;
```

Comment:
I don't see any other code changes in this file. Is this import necessary?

Reply:
> I don't see any other code changes in this file. Is this import necessary?

Nice catch, will be deleted.